packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -20,15 +31,20 @@ const defaultGetCollection = () => null;
  * @return {Array} Words, extracted from the input string.
  */
 function extractWords( input = '' ) {
-	return noCase( input, {
-		splitRegexp: [
-			/([\p{Ll}\p{Lo}\p{N}])([\p{Lu}\p{Lt}])/gu, // One lowercase or digit, followed by one uppercase.
-			/([\p{Lu}\p{Lt}])([\p{Lu}\p{Lt}][\p{Ll}\p{Lo}])/gu, // One uppercase followed by one uppercase and one lowercase.
-		],
-		stripRegexp: /(\p{C}|\p{P}|\p{S})+/giu, // Anything that's not a punctuation, symbol or control/format character.
+	if ( extractedWords.has( input ) ) {
+		return extractedWords.get( input );
+	}
```

Comment:
The two lookups can be squeezed into one `.get` that's tested for `undefined` return value.

Reply:
I agree that's an alternative, but I think the current version is more readable, and I'm not convinced about the performance improvement that would lead to. 

Knowing how browsers usually micro-optimize, there's a chance that the browser will cache the first lookup shortly so the next lookup will be free. 

This is also demonstrated by the [browser tests](https://www.measurethat.net/Benchmarks/Show/18668/0/map-get-vs-map-has-get#latest_results_block) (although I usually don't think those are super reliable), which actually show that more often the `.has()` and `.get()` is faster than just `.get()`.

That being said, I prefer keeping this as-is.